.changeset/nine-apples-change.md

```diff
@@ -0,0 +1,5 @@
+---
+'tests': patch
+---
+
+Convert test scripts and config to TypeScript
```

Comment:
Should be an empty changeset. Changesets are for releases and communicating to users, neither of which we want for this.

Reply:
Done